internal/pkg/agent/application/coordinator/coordinator.go

```diff
@@ -1817,6 +1817,12 @@ func (c *Coordinator) splitModelBetweenManagers(model *component.Model) (runtime
 		case component.OtelRuntimeManager:
 			otelComponents = append(otelComponents, comp)
 		case component.ProcessRuntimeManager:
+			// Hack to fix https://github.com/elastic/elastic-agent/issues/11169
+			// TODO: Remove this after https://github.com/elastic/elastic-agent/issues/10220 is resolved
+			if comp.ID == "prometheus/metrics-monitoring" {
```

Comment:
Create an issue to remove these lines and make it a sub-issue of https://github.com/elastic/elastic-agent/issues/10220, that will make sure we don't forget to come back here and address this.